privatek8s.tf

```diff
@@ -339,7 +339,7 @@ resource "kubernetes_storage_class" "azurefile_csi_premium_retain" {
   provider      = kubernetes.privatek8s
 }
 
-resource "kubernetes_storage_class" "managed_csi_premium_ZRS_retain_public_private" {
+resource "kubernetes_storage_class" "managed_csi_premium_ZRS_retain_private" {
```

Comment:
This change is legit (e.g. using a correct "internal" name for the terraform resource) but it is a refactoring and requires a `moved` block (see https://developer.hashicorp.com/terraform/language/modules/develop/refactoring) to avoid deleting / re-creating the resource in production